acceptance-tests/src/test/resources/features/site/search.feature

```diff
@@ -106,10 +106,18 @@ Feature: Basic search
         When I search for ""test_search""
         Then I should see the search term ""test_search"" on the results page
 
-    Scenario: Blank search terms display the correct results page
+    Scenario: Blank search terms display the full results page
```

Comment:
When reading this scenario together with the next one, if feels to me like this title would be more descriptive if it said '_Searching with no search term provided displays the results page with full, unfiltered result set_'.

In that context, the next one could say '_Navigating to the search page displays the results page with full, unfiltered result set_'.

As it is it's not clear what '_the full results_' page means. Also, in case of the latter scenario, the title doesn't make it immediately clear that it's about navigating without searching.

Would you agree?